static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -508,10 +508,10 @@ const AddViewButton = styled(Button)`
   display: flex;
   color: ${p => p.theme.gray300};
   font-weight: normal;
-  padding: ${space(0.5)} ${space(1)};
-  margin-bottom: 1px;
-  border: none;
-  bottom: -1px;
+  border-radius: 4px 4px 0 0;
```

Comment:
Is the tab border radius different from `theme.borderRadius`?

Reply:
Nope, I just blanked and totally forgot that was a thing. Adding it now!